src/codegen/codegen_neuron_cpp_visitor.cpp

```diff
@@ -286,6 +286,13 @@ void CodegenNeuronCppVisitor::print_neuron_includes() {
 
 
 void CodegenNeuronCppVisitor::print_sdlists_init(bool print_initializers) {
+    printer->add_newline(2);
+    printer->add_line("static void _initlists() {");
+    printer->increase_indent();
+    printer->add_multi_line(R"CODE(
+        static int _first = 1;
+        if (!_first) return;
```

Comment:
Why is this check required? Shouldn't this happen once in `_{mod_name}_reg`?

Reply:
Do you see it in other generated code?

Reply:
No, I don't find any other usage for it so the check should probably go away

Reply:
Thank you, we can add it back if we notice it's needed :)

Reply:
For reference this is now generated with NMODL:
```
    static void _initlists() {
        /* m */
        _slist1[0] = {13, 0};
        /* Dm */
        _dlist1[0] = {16, 0};
        /* h */
        _slist1[1] = {14, 0};
        /* Dh */
        _dlist1[1] = {17, 0};
        /* n */
        _slist1[2] = {15, 0};
        /* Dn */
        _dlist1[2] = {18, 0};
    };
```